Traceback (most recent call last):
  File "/Users/kaihuang1122/Documents/ML/Final/MLP GPT/given_err.py", line 6, in <module>
    from sklearn.model_selection import train_test_split
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/sklearn/__init__.py", line 83, in <module>
    from .base import clone
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/sklearn/base.py", line 19, in <module>
    from .utils import _IS_32BIT
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/sklearn/utils/__init__.py", line 15, in <module>
    from scipy.sparse import issparse
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/scipy/__init__.py", line 57, in <module>
    from ._lib.deprecation import _deprecated
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/scipy/_lib/__init__.py", line 12, in <module>
    from scipy._lib._testutils import PytestTester
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 676, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 579, in module_from_spec
  File "<frozen importlib._bootstrap>", line 558, in _init_module_attrs
  File "<frozen importlib._bootstrap>", line 398, in cached
  File "<frozen importlib._bootstrap_external>", line 570, in _get_cached
  File "<frozen importlib._bootstrap_external>", line 505, in cache_from_source
  File "<frozen importlib._bootstrap_external>", line 128, in _path_join
  File "<frozen importlib._bootstrap_external>", line 128, in <listcomp>
KeyboardInterrupt
